apis/Google.Cloud.Metadata.V1/Google.Cloud.Metadata.V1.IntegrationTests/project.json

```diff
@@ -25,6 +24,7 @@
           "type": "platform"
         }
       }
-    }
+    },
+    "net451": {}
```

Comment:
Yeah, good point. In the future I'll try to separate out something like this. For context, here is where I found the workaround: https://github.com/xunit/xunit/issues/900